src/lib/data/buyables/buyables.ts

```diff
@@ -1,12 +1,12 @@
 import { Bank } from 'oldschooljs';
 
-import { QuestID } from '../../../mahoji/lib/abstracted_commands/questCommand';
 import { chompyHats } from '../../constants';
 import { CombatCannonItemBank } from '../../minions/data/combatConstants';
 import { MinigameName } from '../../settings/settings';
 import { soteSkillRequirements } from '../../skilling/functions/questRequirements';
 import { MUserStats } from '../../structures/MUserStats';
 import { Skills } from '../../types';
+import { QuestID } from '../../util/specificQuests';
```

Comment:
Whats with this change?

Reply:
I thought it would be tidier having the specific quests in its own file rather than in the command file, I can move back that's fine

Reply:
I've now moved all the specific quests back to the questcommand file, so ready for a review again

Reply:
It's a good idea and nice change, but in reality stuff like this can make it harder to review and create annoying merge conflicts. So refactorings are generally best left out, or down on their own separate from a feature update.